### ISSUE TITLE
How to use themes with codemirror 6?

### ISSUE THREAD

**Issue by simPod** [graphql/graphiql#2496]
Hello, how do I use themes with codemirror 6?

I tried this but it does not change visual

```tsx
import GraphiQLComponent from 'graphiql';
import React from 'react';

const graphQLFetcher = ...

export const GraphiQL: React.FC = () => (
  <GraphiQLComponent
    fetcher={graphQLFetcher}
    editorTheme="material"
  />
);
```

**Comment by acao** [comment]
GraphiQL still ships with codemirror 5 - so perhaps try codemirror 5 themes for now?

**Comment by simPod** [comment]
Ah so there's no support for v6 and I have to stay on v5, right?

**Comment by acao** [comment]
For the time being, yes. `codemirror-graphql` ships with support for both, but graphiql sets a hard dependency on 5 for now. 

CC @thomasheyenbrock @imolorhe @yoshiakis I think this would be a good candidate for the 2.x graphiql, and then for 3.x we support either just monaco or both

**Comment by imolorhe** [comment]
Yes I agree. `graphiql` 2.x should use either monaco or CM 6. Considering monaco is more built out, I'll lean towards monaco, especially since CM5 and CM6 APIs are not compatible.

**Comment by ysulyma** [comment]
I'd be interested in implementing CM6 support—is help still needed with this? If so, if there already partial work towards it, or any gotchas to be aware of?

**Comment by dimaMachina** [comment]
closing since graphiql 5 will be powered by monaco-editor
